server/fleet/software_installer.go

```diff
@@ -89,6 +89,8 @@ type SoftwareInstaller struct {
 	InstallScript string `json:"install_script" db:"install_script"`
 	// InstallScriptContentID is the ID of the install script content.
 	InstallScriptContentID uint `json:"-" db:"install_script_content_id"`
+	// UninstallScriptContentID is the ID of the uninstall script content.
+	UninstallScriptContentID uint `json:"-" db:"uninstall_script_content_id"`
```

Comment:
I think we also need UninstallScript here? We need uninstall script visible/editable in the places where we need e.g. InstallScript visible/editable.

Reply:
...yeah, 99.99% sure we do. Going to see if I can comit this to my branch so you can cherry-pick upstream if I did the work correctly.

Reply:
[3ec30ef](https://github.com/fleetdm/fleet/pull/21812/commits/3ec30ef9c13c7830254dae734c706de0862ef050) should do the trick?